applications/games_menu/modules/preset.c

```diff
@@ -469,7 +469,7 @@ void CreateGeneralView(Form *form_ptr)
 		self.async_option = TSU_OptionGroupCreate(form_font, (uint)body_letter_size, 80, body_height_size);
 		TSU_DrawableSetId((Drawable *)self.async_option, ASYNC_CONTROL_ID);
 
-		TSU_OptionGroupAdd(self.async_option, 0, "NONE");
+		TSU_OptionGroupAdd(self.async_option, 0, "TRUE");
```

Comment:
It's NONE if device doesn't support DMA like Serial SD card. It uses emu async and NONE disabled it. TRUE is device support true async DMA.
I don’t like all this, to be honest, but these are already established terms.